docs/sources/tempo/release-notes/v2-3.md

```diff
@@ -26,10 +26,12 @@ This release gives you:
 Tempo 2.3 introduces [vParquet3]({{< relref "../configuration/parquet" >}}), a Parquet version designed to be more compatible with other Parquet implementations, available as a production-ready option.
 This block format improves query performance relative to previous formats.
 
-Read the **Tempo 2.3 blog post** for more examples and details about these improvements.
+Read the [**Tempo 2.3 blog post**](/blog/2023/11/01/grafana-tempo-2.3-release-faster-trace-queries-traceql-upgrades/) for more examples and details about these improvements.
 
 These release notes highlight the most important features and bugfixes. For a complete list, refer to the [Tempo changelog](https://github.com/grafana/tempo/releases).
 
+{{< youtube id="2FWi9_dSBdM?rel=0" >}}
```

Comment:
I see `rel=0` here but not else where.  What's the impact here?

Reply:
As far as I can tell, nothing. The link above was copied from the website for Joe's blog post for 2.3. The others video links were ones I added by grabbing the video IDs from their YouTube URLs. The YouTube URLs didn't have rel=0.